querydsl-apt/src/main/java/com/querydsl/apt/SimpleTypeVisitorAdapter.java

```diff
@@ -13,19 +13,21 @@
  */
 package com.querydsl.apt;
 
-import javax.lang.model.type.TypeMirror;
-import javax.lang.model.util.SimpleTypeVisitor6;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 import java.util.List;
 
+import javax.lang.model.type.IntersectionType;
+import javax.lang.model.type.TypeMirror;
+import javax.lang.model.util.SimpleTypeVisitor6;
+
 /**
- * Converts Java 8 IntersectionType instances into their first bound when visiting
+ * Converts Java 8 {@link IntersectionType} instances into their first bound when visiting
```

Comment:
While visiting the AST, it [flattens the IntersectionType model to just the first bound](https://github.com/querydsl/querydsl/blob/apt-javadocs/querydsl-apt/src/main/java/com/querydsl/apt/SimpleTypeVisitorAdapter.java#L46-L47).
So `Number & Comparable` would just be `Number`.